release/lib/utils.ts

```diff
@@ -0,0 +1,181 @@
+import * as path from 'path';
+
+import * as execasync from 'execa';
+import * as fs from 'fs-extra';
+import pRetry from 'p-retry';
+
+import { Package, Context } from '../@types/custom';
+import { ListrTaskWrapper } from 'listr';
+import { Observable, Subscriber } from 'rxjs';
+
+export const logFilename = `release.log`;
+export const REPOSITORY_SLUG = 'webhintio/hint';
+export const REPOSITORY_URL = `https://github.com/${REPOSITORY_SLUG}`;
+
+/**
+ * The output stream to `release.log`
+ */
+// TODO: We should wait for the 'open' event to be sure everything is fine
```

Comment:
Was this something you wanted to do before merging?

Reply:
not really, it complicates the code quite a bit and we take long enough to have the stream ready before the first use... I'll just leave the comment here so at least we know I was aware in case something happens.